project-base/.ci/deploy-to-google-cloud.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh -ex
+
+# Login to Docker Hub for pushing images into register
+echo ${DOCKER_PASSWORD} | docker login --username ${DOCKER_USERNAME} --password-stdin
+
+# Create unique docker image tag with commit hash
+DOCKER_IMAGE_TAG=production-commit-${GIT_COMMIT}
+
+## Docker image for application php-fpm container
+docker image pull ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG} || (
```

Comment:
seems like we stopped to compile microservices so this looks like script with cookbook just for project-base, is that right?

Reply:
They are not because in project base context there does not exist source codes to microservices. 

Yeah, its for project-base usage. We do not expect someone to develop projects in monorepo.